README.md

```diff
@@ -18,39 +20,27 @@
 
 ## Getting Started
 
-**This is an example of how you may give instructions on setting up your project locally.**
```

Comment:
- [ ] Please add directions on how to clone and run the app locally.

Reply:
Ok @dasileker Thank you, will do